src/types/requestResponses.ts

```diff
@@ -4,15 +4,17 @@ interface BaseNotificationResponse {
   trackId?: string;
 }
 
-export type SendNotificationTypeRequest = {
-  /**
-   * @example "Made donation"
-   */
-  eventName: string;
+export interface SendNotificationRequest {
   /**
    * @example "Should be unique"
    */
   trackId?: string;
```

Comment:
@aminlatifi 
I moved `trackId` to this interface then changed `SendNotificationTypeRequestBulkItem` to extend this and override `trackId` and make it `required`